code/src/components/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import moment from 'moment'
+
+import styled from 'styled-components'
+ 
+const Header = () => {
+  const currentDate = moment().format("dddd, MMM Do");
+  const currentTime = moment().format("LT");
```

Comment:
I don't think Moment is used anymore, perhaps look into different solutions for showing date and time. :)

Reply:
She is using it in line 14 😄